test/unit/inabox/test-inabox-viewport.js

```diff
@@ -236,7 +239,8 @@ describes.fakeWin('inabox-viewport', {amp: {}}, env => {
             width: 1000,
             height: 2000,
           },
-        }));
+        }),
+        false, true);
```

Comment:
These would be clearer as `/*varName=*/ false, /*otherVarName=*/ true`

Reply:
Done.